Tests/appstoreconnect-cliTests/Models/Bundle+Tests.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Files
+import Foundation
+
+extension Folder {
+
+    // swiftlint:disable force_try
+    static let tests: Folder = try! Folder(
+        path: URL(fileURLWithPath: #file)
+            .deletingLastPathComponent()
+            .appendingPathComponent("Fixtures.bundle").path
+    )
```

Comment:
I was trying to use something like 
```
private final class BundleTag {}
static let tests = Bundle(for: BundleTag.self)

tests.url(forResource: named, withExtension: "json", subdirectory: "Fixtures.bundle")
```
But couldn't make it work, it keeps telling me can't find bundle 😢

Reply:
🤔 resource bundles and SPM are still not friends. Didn't realise this extended to tests. Should be some good news this week with Swift 5.3 and Xcode 12 though
https://developer.apple.com/videos/play/wwdc2020/10169/

Reply:
@adamjcampbell Yes but I tried this mainly because I saw @orj did the same thing in `Apple store connect SDK` which itself is also a SPM project. But just can't make this work here 😢

Reply:
With Swift 5.3 you should be able to use `Bundle.module` for this.